src/Talis/Persona/Client/Base.php

```diff
@@ -55,40 +57,55 @@
     private $defaultTtl;
 
     /**
+     * @var string
+     */
+    private $appUserAgent;
+
+    /**
+     * @var string
+     */
+    private $phpVersion;
+
+    /**
      * Constructor
      *
+     * @param string $appUserAgent Consuming application user agent string @since 2.0.0
+     *      examples: rl/5.2 (php/5.3), rl, rl/5, rl/5.2(php/5.3; linux/2.5)
```

Comment:
How come?
On 15 Apr 2016 8:05 p.m., "Chris Clarke" <notifications@github.com> wrote:

> In src/Talis/Persona/Client/Base.php
> <https://github.com/talis/persona-php-client/pull/27#discussion_r59925480>
> :
>
> >       * Constructor
> >       *
> > +     * @param string $appUserAgent Consuming application user agent string @since 2.0.0
> > +     *      examples: rl/5.2 (php/5.3), rl, rl/5, rl/5.2(php/5.3; linux/2.5)
>
> @foomatty <https://github.com/foomatty> wants us to take out the PHP/Node
> ver...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/talis/persona-php-client/pull/27/files/0068c087213f968c6fa02bf4a9e01d8ac9863c16#r59925480>
>